src/index.html

```diff
@@ -1,22 +1,422 @@
+                      src="./images/categories/Image-3.png"
+                      alt="Earphones"
+                    />
+                  </a>
+                </div>
+
+                <h3 class="category__title">Earphones</h3>
+              </article>
+
+              <article class="category">
+                <div class="category__photos">
+                  <a
+                    href="#"
+                    class="category__link category__link--square"
+                  >
+                    <img
+                      class="category__photo"
+                      src="./images/categories/Image-4.png"
+                      alt="Smart Home"
+                    />
+                  </a>
+
+                  <a
+                    href="#"
+                    class="category__link category__link--square"
+                  >
+                    <img
+                      class="category__photo"
+                      src="./images/categories/Image-5.png"
+                      alt="Smart Home"
+                    />
+                  </a>
+                </div>
+
+                <h3 class="category__title">Smart Home</h3>
+              </article>
+
+              <button class="button">View all</button>
+            </div>
+          </section>
+        </div>
+
+        <div>
+          <div class="empty"></div>
+
+          <div class="how-to-buy--color">
+            <section
+              class="how-to-buy container"
+              id="how-to-buy"
+            >
+              <h2
+                class="how-to-buy__title section-title section-title--align--left"
+              >
+                Timeless, for 50 years and counting
+              </h2>
+
+              <div class="how-to-buy__content">
+                <p class="how-to-buy__description">
+                  Bang & Olufsen is expanding its Recreated Classics Program
+                  with the launch of Beosystems – a limited edition music system
+                  that transcends time by bridging the gap between one of our
+                  iconic designs from 1972 and today’s cutting-edge digital
+                  technology.
+                </p>
+              </div>
+            </section>
+          </div>
+        </div>
+
+        <div class="container">
+          <section
+            class="contacts"
+            id="contacts"
+          >
+            <h2 class="section-title section-title--align--left">Contact us</h2>
+
+            <section
+              class="contact-us"
+              id="contact-us"
+            >
+              <form
+                class="contact-us__form"
+                onsubmit="this.reset(); return false;"
+              >
+                <input
+                  type="text"
+                  placeholder="Name"
+                  class="contact-us__input"
+                  required
+                />
+                <input
+                  type="email"
+                  placeholder="Email"
```

Comment:
Placeholders should give examples of what should be added, to make the expected format clear